### PR TITLE
*/*: drop more dead (prefix) target cruft

### PR DESCRIPTION
**Pull request by thesamesam**
Signed-off-by: Sam James <sam@gentoo.org>

### CODE REVIEW

File: eclass/xorg-3.eclass [hunk 1]
```diff
@@ -273,19 +273,7 @@ xorg-3_src_unpack() {
...
-			;;
-		*)
-			# elibtoolize required for BSD
-			[[ ${XORG_EAUTORECONF} != no && ( -e "./configure.ac" || -e "./configure.in" ) ]] \
-				&& XORG_EAUTORECONF=yes
-			;;
-	esac
-
+	[[ ${XORG_EAUTORECONF} != no && ( -e "./configure.ac" || -e "./configure.in" ) ]] && XORG_EAUTORECONF=yes
 	[[ ${XORG_EAUTORECONF} != no ]] && eautoreconf
```
Comment (SoapGentoo) [inline eclass/xorg-3.eclass:277]: unify these two lines into one common block?

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: eclass/xorg-3.eclass (hunk 1 above)
Comment (thesamesam) [reply eclass/xorg-3.eclass:277]: I don't think this even made sense before..

The problem is, the current/previous logic is:
* if `XORG_EAUTORECONF` is explicitly no, just elibtoolize.
* if `XORG_EAUTORECONF` is yes/unset, automagic it based on configure.{ac,in} presence
* if `XORG_EAUTORECONF` is unset, autoreconf

I don't see the point in the automagic, so let me clean it up.

Comment (SoapGentoo) [reply eclass/xorg-3.eclass]: ...it's dangerous in fact. How can you eautoreconf without ascertaining you actually have the deps?

**Review by grobian — COMMENTED** [review]
looks good, thanks!

**Pull request closed without merging**